src/module/Entity/src/Entity/Form/ModulePageForm.php

```diff
@@ -32,21 +32,9 @@ function __construct(LicenseInterface $license)
         $select->setLabel('Select an icon');
         $select->setAttribute('id', 'icon');
         $select->setValueOptions(array(
-                'book-open' => 'Open book',
-                'book' => 'Closed book',
-                'bookmark' => 'Bookmark',
-                'play-button' => 'Play',
-                'google-maps' => 'Marker',
-                'film' =>'Film',
-                'circle-question-mark' => 'Question mark (circle)',
-                'circle-exclamation-mark' => 'Exclamation mark (circle)',
-                'circle-info' => 'Info (circle)',
-                'pencil' => 'Pencil',
-                'file' => 'File',
-                'notes' => 'Notes',
-                'stats' => 'Stats',
-                'compass' => 'Compass',
-                'lightbulb' => 'Lightbulb'
+                '' => 'Explanation',
```

Comment:
@inyono Yes, that's correct. Bit of a UX discussion here. When I talked with Michael about this, we wanted to make it clear to the user, when to use the "no icon" option, espacially destinguishing it from videos and exercises.